packages/components/src/HeaderBar/HeaderBar.component.js

```diff
@@ -41,21 +41,22 @@ function Logo({ isFull, renderers, ...props }) {
 				bsStyle="link"
 				className={actionClassName}
 				hideLabel
-				label="Go to Portal"
+				label={t('GO_PORTAL')}
```

Comment:
t('GO_PORTAL', 'Go to Portal') with a defaultTranslate function which support this second argument